main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	fieldsParameter := flag.String("f", "", "Choose field to print")
+	flag.Parse()
+
+	content, readAllErr := ioutil.ReadAll(os.Stdin)
+	if readAllErr != nil {
+		fmt.Fprintln(os.Stderr, readAllErr)
+		os.Exit(1)
+	}
+
+	if len(content) <= 0 {
+		return
+	}
+
+	input := strings.Split(*fieldsParameter, ",")
+	inputSplit := make([][]string, len(input))
+	for i := 0; i < len(input); i++ {
+		inputSplit[i] = strings.Split(input[i], "-")
+	}
+
+	contentLineSplit := bytes.Split(content, []byte{'\n'})
+	data := make([][][]byte, len(contentLineSplit))
+
+	for i := 0; i < len(contentLineSplit); i++ {
+		data[i] = bytes.Split(contentLineSplit[i], []byte{9})
+		// fmt.Println(data[i])
+	}
+
+	counter := 0
+	for a := range data {
+		if len(data[a]) > counter {
+			counter = len(data[a])
+		}
+	}
+
+	/*
+		for a := range data {
+			for i := 0; i < counter; i++ {
+				if len(data[a][i]) < 1 {
+					data[a][i] = []byte("")
+				}
+			}
+		} */
+
+	c := len(data) - 1
+	var actualRange int
+
+	if len(data[c][0]) > 0 {
+		actualRange = len(data)
+	} else {
+		actualRange = len(data) - 1
+	}
+
+	// fmt.Println(len(data)) // Lines
+
+	for i := 0; i < actualRange; i++ {
+		for j := range input {
+			if len(inputSplit[j]) == 1 {
+				k, err := strconv.Atoi(inputSplit[j][0])
```

Comment:
The larger the input, the more often you're parsing -f's ints. Do this only once to increase efficiency.